ansys/dpf/core/collection.py

```diff
@@ -329,7 +329,8 @@ def __getitem__(self, index):
         if not self_len:
             raise IndexError("This collection contains no items")
         if index >= self_len:
-            raise IndexError(f"This collection contains only {self_len} entrie(s)")
+            word = "entry" if self_len == 1 else "entries"
+            raise IndexError(f"This collection contains only {self_len} {word}")
```

Comment:
Error messages should be complete sentences, ending in a period. This can be addressed in future PRs over time. I'm not sure I fully comprehend what the error in line 333 would look like. If the word was ball, would the message be: This collection contains only ball."? Perhaps it would help to have the word item: This collection contains only one item, ball.

Reply:
@PipKat thank you, I will add the period.
As for the resulting sentence it would be: 
- if only one "item" -> "This collection contains only 1 entry."
- if more -> "This collection contains only X entries."
This error message tells the user it is asking for more than exists.

Reply:
@PipKat Thank you for the link!